src/sort/vec.rs

```diff
@@ -69,21 +83,8 @@ impl Sort for VecSort {
         result
     }
 
-    fn canonicalize(&self, value: &mut Value, unionfind: &UnionFind) -> bool {
```

Comment:
Why is this getting backed out? Don't we still want to canonicalize vectors?

Reply:
Oof, good catch, it shouldn't be backed out.
We really need better testing for containers